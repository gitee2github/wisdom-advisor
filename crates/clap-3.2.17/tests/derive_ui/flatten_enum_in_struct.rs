#[derive(clap::Parser)]
struct Opt {
    #[clap(flatten)]
    sub: SubCmd,
}

#[derive(clap::Parser)]
enum SubCmd {}

fn main() {}
