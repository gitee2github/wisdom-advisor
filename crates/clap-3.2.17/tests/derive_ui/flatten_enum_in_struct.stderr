error[E0277]: the trait bound `SubCmd: clap::Args` is not satisfied
   --> tests/derive_ui/flatten_enum_in_struct.rs:3:12
    |
3   |     #[clap(flatten)]
    |            ^^^^^^^ the trait `clap::Args` is not implemented for `SubCmd`
    |
note: required by `augment_args`
   --> src/derive.rs
    |
    |     fn augment_args(cmd: Command<'_>) -> Command<'_>;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0277]: the trait bound `SubCmd: clap::Args` is not satisfied
   --> tests/derive_ui/flatten_enum_in_struct.rs:3:12
    |
3   |     #[clap(flatten)]
    |            ^^^^^^^ the trait `clap::Args` is not implemented for `SubCmd`
    |
note: required by `augment_args_for_update`
   --> src/derive.rs
    |
    |     fn augment_args_for_update(cmd: Command<'_>) -> Command<'_>;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
